docs/docs/overview/performance.md

```diff
@@ -6,103 +6,178 @@ This section analyzes the performance of Constellation.
 
-Services: Constellation has lower latencies than AKS and GKE except for service creation on AKS.
+##### Pod-to-Pod
 
-![API Latency - Deployments](../_media/benchmark_api_dpl.png)
+In this scenario, the client Pod connects directly to the server pod in its IP address.
 
-Deployments: Constellation has the lowest latency for all cases except for scaling deployments on GKE and creating deployments on AKS.
+![Pod2Pod concept](../_media/benchmark_p2p_concept.webp)
 
-#### Network
 
-There are two main indicators for network performance: intra-node and inter-node transmission speed.
-K-Bench provides benchmark tests for both, configured as `dp_network_internode` and `dp_network_intranode`. The tests use [`iperf`](https://iperf.fr/) to measure the bandwidth available.
+The results for "Pod-to-Pod" TCP are as follows:
 
-##### Inter-node
+![Network Pod2Pod TCP benchmark graph](../_media/benchmark_net_p2p_tcp.png)
 
-Inter-node communication is the network transmission between different Kubernetes nodes.
+The results for "Pod-to-Pod" UDP are as follows:
 
-The first test (`dp_network_internode`) measures the throughput between nodes. Constellation has an inter-node throughput of around 816 Mbps on Azure to 872 Mbps on GCP. While that's faster than the average throughput of AKS at 577 Mbps, GKE provides faster networking at 9.55 Gbps.
-The difference can largely be attributed to Constellation's [network encryption](../architecture/networking.md) that protects data in-transit.
+![Network Pod2Pod UDP benchmark graph](../_media/benchmark_net_p2p_udp.png)
 
-##### Intra-node
+##### Pod-to-Service
 
-Intra-node communication happens between pods running on the same node.
-The connections directly pass through the node's OS layer and never hit the network.
-The benchmark evaluates how the [Constellation's node OS image](../architecture/images.md) and runtime encryption influence the throughput.
+Tn this scenario, the client Pod connects to the server Pod via a ClusterIP service. This is more relevant to real-world use cases.
 
-Constellation's bandwidth for both sending and receiving is at 31 Gbps on Azure and 22 Gbps on GCP. AKS achieves 26 Gbps and GKE achieves about 27 Gbps in the tests.
+![Pod2Pod concept](../_media/benchmark_p2svc_concept.webp)
 
-![](../_media/benchmark_net.png)
+The results for “Pod-to-Service” TCP are as follows:
 
+![Network Pod2SVC TCP benchmark graph](../_media/benchmark_net_p2svc_tcp.png)
+
+The results for “Pod-to-Service” UDP are as follows:
+
+![Network Pod2SVC TCP benchmark graph](../_media/benchmark_net_p2svc_udp.png)
+
+Comparing Constellation on GCP with GKE, Constellation has 58% less TCP bandwidth less but a similar UDP bandwidth.
+Constellation on Azure isn't directly comparable as the Confidential VM types in Azure have only 10% of the regular VM types' bandwidth.
+Constellation's network encryption adds almost no overhead on top of that limitation.
```

Comment:
Isn't the overhead around 30%?

Reply:
It should be accurate now. ~58% loss on GCP, ~15% loss on Azure